projects/example-app/src/app/app.routing.ts

```diff
@@ -1,15 +1,19 @@
-import { NgModule } from '@angular/core';
-import { Routes, RouterModule } from '@angular/router';
+import { Routes } from '@angular/router';
```

Comment:
Can you rename the `*.routing.ts` files to `*.routes.ts` please?
This keeps things more inline with the Angular documentation/recommendation.

Reply:
Done